### PR TITLE
Feature filter against feature-invoke-service-POST

### PR DESCRIPTION
**Pull request by Vivek205**

### CODE REVIEW

File: src/components/AiMarketplace/MainSection/Filter/index.js [hunk 1]
```diff
@@ -1,50 +1,50 @@
...
     const latestPagination = { ...pagination, ...defaultPaginationParameters, s: filterParamters[name], q: value };
     await updatePagination(latestPagination);
-    await fetchService(latestPagination);
-    setActiveFilterItem({
-      ...activeFilterItem,
-      [name]: value,
-    });
+    await fetchService(latestPagination, filterObj);
+    setActiveFilterItem(currentActiveFilterData);
+    stopLoader();
```
Comment (chetankothari) [inline src/components/AiMarketplace/MainSection/Filter/index.js]: All the above could be moved into an action.

File: src/components/AiMarketplace/MainSection/Filter/index.js [hunk 2]
```diff
@@ -1,50 +1,50 @@
...
   };
 
   const handleFilterReset = async () => {
```
Comment (chetankothari) [inline src/components/AiMarketplace/MainSection/Filter/index.js:44]: This could be converted into an action.

**Review by chetankothari — APPROVED** [review]
Minor suggestions, looks good otherwise.

File: src/components/AiMarketplace/MainSection/Filter/index.js (hunk 1 above)
Comment (Vivek205) [reply src/components/AiMarketplace/MainSection/Filter/index.js]: [done](https://github.com/Vivek205/refactored-dapp/pull/39/commits/711d01b309dcc071bb928d67343b72a20bb073f5)

File: src/components/AiMarketplace/MainSection/Filter/index.js (hunk 2 above)
Comment (Vivek205) [reply src/components/AiMarketplace/MainSection/Filter/index.js:44]: [done](https://github.com/Vivek205/refactored-dapp/pull/39/commits/711d01b309dcc071bb928d67343b72a20bb073f5)

**Pull request closed without merging**